regression-tests/convert-FITS_+_generate_fit_quality/convert-FITS_+_generate_fit_quality.sh

```diff
@@ -2,20 +2,27 @@
 
 die() { echo "FATAL ERROR IN CONVERT FITS: $@" >&2; exit 1;}
 
+abs(){
+	return 'scale=8;sqrt($1 - $2) ^ 2)' | bc
+}
 
 mkdir -p $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.tmp
 mkdir -p $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out
 $SPARCFIRE_HOME/scripts/SpArcFiRe -convert-FITS $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.in/ $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.tmp $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out -generateFitQuality 1 > fit_quality_test.txt 2> fit_quality_err.txt
 TEST_RESULT=1
 
-diff <(cut -f1-39,42-150 -d$','  $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out/galaxy.csv) <(cut -f1-39,42-150 -d$','  $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/test.out/galaxy.csv) > comp.txt
+sed 's/[^0-9.EefF-]/ /g' $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/G.out/galaxy.csv > /tmp/sPaRCfIrE_TC1
+sed 's/[^0-9.EefF-]/ /g' $SPARCFIRE_HOME/regression-tests/convert-FITS_+_generate_fit_quality/test_data/test.out/galaxy.csv > /tmp/sPaRCfIrE_TC2
 
-if ! [ -s "comp.txt" ]; then
+paste /tmp/sPaRCfIrE_TC1 /tmp/sPaRCfIrE_TC2 > /tmp/sPaRCfIrE_TCP1
+
+awk '{for(i=1;i<=NF;i++) if(abs($1 $2) > 1.0e-6) print "different"}' > /tmp/sPaRCfIrE_TCP1
```

Comment:
On what system have you ever seen "abs" as a built-in function to awk? This won't work; plus you've put ($1 $2) without a minus sign between them.

To make it work, it's easy: add the following before the rest of the awk code:

function abs(a,b) { if(a>b) return a-b; else return b-a;}